env.sh

```diff
@@ -15,7 +15,7 @@ LOCAL_JWT_SIGNING=TRUE
 DOCKER_COMPOSE=TRUE
 #DOCKER_COMPOSE=FALSE
 
-DOCKER_IMAGE="digitalasset/daml-sdk:1.5.0"
+DOCKER_IMAGE="digitalasset/daml-sdk:1.10.0"
```

Comment:
why is only this line changing? I would have expected a bunch of other references like `daml.yaml` to change as well.

Reply:
The upgrade to 1.10 was done in a previous PR. This corrects for the Docker-compose configuration which was missed.